python/frontend/cirrus/cirrus/HashingTrick.py

```diff
@@ -0,0 +1,48 @@
+# Apply the hashing trick to specified columns
+
+import json
+import time
+import boto3
+from collections import deque
+from serialization import LambdaThread, get_all_keys
+from threading import Thread
+import random
+
+class HashingThread(LambdaThread):
+    def __init__(self, s3_bucket_input, s3_bucket_output, s3_key, columns, N, invocation):
+        Thread.__init__(self)
+        self.d = {
```

Comment:
I think that the dictionary makes the most sense here, given that it will be passed directly to the Lambda function, but feel free to correct me if I'm wrong here